packages/test/test-service-load/src/loadTestDataStore.ts

```diff
@@ -27,23 +27,25 @@ export interface IRunConfig {
     randEng: random.Engine,
 }
 
-export interface ILoadTest {
+export type ILoadTest = DataObject & {
```

Comment:
why do we need DataObject here? is it just for request? if so, we should just extend IFluidRouter